GrpcClient/GrpcRepo.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Runtime.InteropServices.JavaScript;
+using DTOs.Shift;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Net.Client;
+
+
+namespace GrpcClient;
+
+public class GrpcRepo
+{
+    public async Task<DTOs.EmployeeDTO> CreateEmployee(DTOs.NewEmployeeDTO newEmployeeDto)
+    {
+        //create channel for connection to JAVA
+        using var channel = GrpcChannel.ForAddress("http://192.168.125.143:50051"); //TODO the port might change
+        //create client for employee entity, "EmployeeClient" method is auto-genereted               
+        var client = new Employee.EmployeeClient(channel);
+        //reply - "AddSingleEmployee(newEmployeeDto)" comes from .proto file + need to convert the og-DTO to .proto-DTO
+        var reply =  client.AddSingleEmployee(new NewEmployeeDTO
+        {
+            FirstName = newEmployeeDto.FirstName,
+            LastName = newEmployeeDto.LastName,
+            WorkingNumber = UInt32.CreateChecked(newEmployeeDto.WorkingNumber),
+            Email = newEmployeeDto.Email,
+            Password = newEmployeeDto.Password
+        }); //TODO await maybe
```

Comment:
why await here?

Reply:
idk but it is not needed YET i hope it will stay like that

Reply:
@kvecko121 delete the comment